test/e2e/common/util.go

```diff
@@ -63,6 +65,45 @@ var CommonImageWhiteList = sets.NewString(
 	imageutils.GetE2EImage(imageutils.Net),
 )
 
+var testImages = struct {
```

Comment:
hidden down in kubectl I maybe understand this, but pulling this pattern up out of the tests that directly reference the images feels like it's going to be prone to breakage

could you share a function with the go template that calls GetE2EImage instead?